deployment/ansible/templates/epoch.yaml.j2

```diff
@@ -4,10 +4,11 @@ sync:
 
 peers:
 {% for host in groups[hosts_group] | difference([inventory_hostname]) %}
+{% set public_ip = hostvars[host]['ansible_ssh_host']|default(hostvars[host]['ansible_host']) -%}
```

Comment:
This looks a duplicate of public_ip var in deploy.yml?

Reply:
Yes, however if the playbook is run with `--limit` option the [`set_fact`](https://github.com/aeternity/epoch/pull/939/files#diff-2ad1947fe9b869ffd23a6a86872e65e0L30) is executed only on the limited set of hosts specified.

Having this in the template in the `for` loop makes sure it get's the correct IP of all the hosts in the group/env.

Reply:
Moreover, we can't use the [`public_ip`](https://github.com/aeternity/epoch/pull/939/files#diff-2ad1947fe9b869ffd23a6a86872e65e0R42) var in the template, but `set_fact` that was used before.

Reply:
As the PR title suggest, this is fixing deployments to limited number of hosts (e.g. 1 host), otherwise if you always deploy/limit to the full group/env, the old version (before this PR) still works as expected.